contracts/pnpm-lock.yaml

```diff
@@ -1,228 +1,172 @@
-lockfileVersion: '6.0'
-
-settings:
-  autoInstallPeers: true
-  excludeLinksFromLockfile: false
+lockfileVersion: 5.4
```

Comment:
Please update your pnpm and do not commit this lock file downgrade

Reply:
yes will do locally. this PR won't get merged to develop. This should not affect how this image will work on the off-chain side of things for CL nodes